forge-gui/res/cardsfolder/upcoming/the_caves_of_androzani.txt

```diff
@@ -0,0 +1,14 @@
+Name:The Caves of Androzani
+ManaCost:3 W
+Types:Enchantment Saga
+K:Saga:4:DBStunCounter,DBChooseCard,DBChooseCard,DBSearch
+SVar:DBStunCounter:DB$ PutCounter | ValidTgts$ Creature.tapped | TgtPrompt$ Select un two two target tapped creature | CounterType$ STUN | CounterNum$ 2 | TargetMin$ 0 | TargetMax$ 2 |  SpellDescription$ Put two stun counters on each of up to two target tapped creatures.
+SVar:DBChooseCard:DB$ ChooseCard | Choices$ Permanent.nonSaga+HasCounters | ChoiceTitle$ Choose any number of non-Saga permanents | MinAmount$ 0 | ChoiceZone$ Battlefield | Amount$ X | SubAbility$ DBAdditionalCounter | SpellDescription$ For each non-Saga permanent, choose a counter on it. You may put an additional counter of that kind on that permanent.
```

Comment:
I think this is fine for now... eventually there will be a refactor involving actually choosing the counters, but it seems like just busy work for one weird line of text at this point. Probably a better prompt will smooth it over for now – "Choose any number of non-Saga permanents with counters" might do the trick?